package.json

```diff
@@ -51,6 +52,10 @@
     "webpack-merge": "^4.1.5"
   },
   "dependencies": {
+    "@babel/polyfill": "^7.2.5",
+    "engage-onsite-sdk-js": "git+ssh://git@github.com:chaordic/engage-onsite-sdk-js.git#v0.0.0",
```

Comment:
se esse projeto for buildado, todas essas paradas podem ser devDependencies

Reply:
a parte de build vai ser feita somente para o npm, então é preferível deixar nesse formato